src/LayerEditor/ui/dialogs/layers/split_layer.py

```diff
@@ -24,7 +25,7 @@ def __init__(self, min, max, copy_block, parent=None):
 
         self.image = QtWidgets.QLabel(self)
```

Comment:
Is it possible to reuse this check indicator as a QLabel descendant, we should introduce few of such classes used across the LayerEditor.